### ISSUE TITLE
is it after build?

### ISSUE THREAD

**Issue by talforsher** [mickySignNow/SignPlusApp#1]
is it after build?
if yes, is it needed to build before push? I guess not. and how it looks before build?
if not, please explain how it should look after build.

**Comment by mickySignNow** [comment]
Yes it is after build.. the reason it doesnt run is because you need to run
the app on a specifix port.
Run in terminal:
Flutter run -d web --web-port 5000

בתאריך יום ה׳, 3 בדצמ׳ 2020, 13:54, מאת talforsher ‏<
notifications@github.com>:

**Comment by mickySignNow** [comment]
Later on we will connect ot to a domain and it will use its port.

בתאריך יום ה׳, 3 בדצמ׳ 2020, 16:25, מאת מיקי קרויטרו ‏<micky@signow.org>:
